src/components/tuits/tuit.js

```diff
@@ -47,14 +50,42 @@ const Tuit = ({tuit, deleteTuit, likeTuit, dislikeTuit}) => {
                     <i onClick={() => deleteTuit(tuit._id)}
                        className="fas fa-remove tuit-button fa-2x fa-pull-right"/>
```

Comment:
Update to have the pencil, green checkmark, and red trash can as shown in main branch (may be fixed during selection during merge?)